examples/Huggingface_Transformers/Image_classification_docker/HF-only/requirements.txt

```diff
@@ -0,0 +1,2 @@
+#Assuming that this `requirements.txt` file is run by a Cloud instance that already comes with PyTorch and Torchserve preinstalled
+transformers
```

Comment:
We have a developer.txt that folks can leverage https://github.com/pytorch/serve/blob/master/requirements/developer.txt

We already have way too many requirements.txt and I don't want to add more

Reply:
Yes that would do too as it contains the `transformers` dependency. Would add asap

Reply:
@msaroufim I am getting an error while trying to use it, it is solved when I remove the first line from developer.txt (i.e `-r common.txt`). 

The Torchserve logs show the following in the file `logs/ts_logs.log` 
![image](https://user-images.githubusercontent.com/42506819/187613509-95acab29-3530-48fd-afb7-90d2ff1f4f58.png) .

Should we remove the `-r common.txt` line from developer.txt?